pkg/scaffold/helm/gopkgtoml.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helm
+
+import (
+	"github.com/operator-framework/operator-sdk/pkg/scaffold"
+	"github.com/operator-framework/operator-sdk/pkg/scaffold/input"
+)
+
+// GopkgToml - the Gopkg.toml file for a hybrid operator
+type GopkgToml struct {
+	input.Input
+}
+
+func (s *GopkgToml) GetInput() (input.Input, error) {
+	if s.Path == "" {
+		s.Path = scaffold.GopkgTomlFile
+	}
+	s.TemplateBody = gopkgTomlTmpl
+	return s.Input, nil
+}
+
+const gopkgTomlTmpl = `[[constraint]]
+  name = "github.com/operator-framework/operator-sdk"
+  # The version rule is used for a specific release and the master branch for in between releases.
+  branch = "master" #osdk_branch_annotation
+  # version = "=v0.3.0" #osdk_version_annotation
```

Comment:
I just remembered that we need to update this scaffold for each release as well now.

@joelanford Can you please update the [release guide section](https://github.com/operator-framework/operator-sdk/blob/master/doc/dev/release.md#1-create-a-pr-for-release-version-and-changelogmd-updates) for the version update as well.

We need to bump both [`pkg/scaffold/ansible/gopkgtoml.go`](https://github.com/operator-framework/operator-sdk/blob/master/pkg/scaffold/ansible/gopkgtoml.go#L39) and this file `pkg/scaffold/helm/gopkgtoml.go`

@estroz We just need to add checks for these two files in the release.sh script to ensure the versions are set correctly at release time as well right? Similar to the existing checks:
https://github.com/operator-framework/operator-sdk/blob/master/release.sh#L38-L41

Reply:
@estroz scratch that. I just saw @joelanford already added the check for this file down in release.sh.